docs/components/gantry/_index.md

```diff
@@ -363,7 +366,8 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/_modules
 ```python
 my_gantry = Gantry.from_robot(robot=robot, name="my_gantry")
 
-# Stop all motion of the gantry. It is assumed that the gantry stops immediately.
+# Stop all motion of the gantry. It is assumed that the gantry stops
+# immediately.
```

Comment:
Is there a way to make the two lines more evenly distributed automatically? Here and some other places, there are just one or two words on the second line and I wish it'd automatically do something like 
```suggestion
# Stop all motion of the gantry. It is assumed that the
# gantry stops immediately.
```

Reply:
I don't mind accepting this change but that's more personal preference, the only linting rule is for lines not to exceed 80 characters. Just probably won't go looking for other instances right now.